policyengine_us/variables/gov/states/mt/tax/income/deductions/cdcc/mt_child_dependent_care_expense_deduction.py

```diff
@@ -3,24 +3,26 @@
 
 class mt_child_dependent_care_expense_deduction(Variable):
     value_type = float
-    entity = TaxUnit
+    entity = Person
     label = "Montana child dependent care expense deduction"
     unit = USD
     definition_period = YEAR
     reference = "https://casetext.com/statute/montana-code/title-15-taxation/chapter-30-individual-income-tax/part-21-rate-and-general-provisions/section-15-30-2131-repealed-effective-112024-temporary-deductions-allowed-in-computing-net-income"
     defined_for = StateCode.MT
 
-    def formula(tax_unit, period, parameters):
-        eligible_children = tax_unit(
+    def formula(person, period, parameters):
+        eligible_children = person.tax_unit(
```

Comment:
Since tax_unit used multiple times. Also keeping comments above to minimize wrapping.
```suggestion
        tax_unit = person.tax_unit
        # Line 1.
        eligible_children = tax_unit(
```